doc/interop-test-descriptions.md

```diff
@@ -130,11 +138,39 @@ Procedure:
       zero and comparing the entire response message against a golden response
 
 
- 2. Client calls UnaryCall with:
+### server_compressed_unary
+
+This test verifies the server can compress unary messages. It sends two unary
+requests for a COMPRESSABLE payload type, expecting the server response to be
+compressed or not according to the `request_compressed_response` boolean.
+
+Whether compression was actually performed is determined by the compression bit
+in the response's message flags.
+
+
+Server features:
+* [UnaryCall][]
+* [Compressable Payload][]
+* [Compressed Response][]
+
+Procedure:
+ 1. Client calls UnaryCall with:
+
     ```
     {
-      request_compressed_response: bool
-      response_type: UNCOMPRESSABLE
+      request_compressed_response: true
```

Comment:
naming:  `response_size` means   send me a response of given size.  With that logic, `request_compressed_response` should be   just `response_compressed`  the "request_" prefix is extraneous.